src/app/common/server-calls/fetchVersionNumber.ts

```diff
@@ -0,0 +1,5 @@
+export async function fetchVersionNumber(): Promise<string> {
+  const response = await fetch('https://api.github.com/repos/hirosystems/explorer/releases');
```

Comment:
This seems like the wrong behavior? This code will always report the latest release, even if an older version of the code is deployed. The version should be statically populated when a release is tagged IMO.